gravitee-apim-console-webui/src/management/api/documentation-v4/documentation-new-page/api-documentation-v4-new-page.component.ts

```diff
@@ -36,6 +36,7 @@ export class ApiDocumentationV4NewPageComponent implements OnInit, OnDestroy {
   pageTitle = 'Add new page';
   source: 'FILL' | 'IMPORT' | 'EXTERNAL' = 'FILL';
   content = '';
+  preview = false;
```

Comment:
Should the preview be default as true?

Reply:
it was @anthonyjgatti's preference to make it disabled by default. 🤷

Reply:
Ok. We can keep it disabled for now and maybe @anthonyjgatti will be convinced otherwise in seeing it :P